### DISCUSSION TITLE
Auto ignore Style/IfUnlessModifier inside transactions

### DISCUSSION THREAD

**Discussion by Galaxy83** [rubocop/rubocop#10595] [Maintenance]
Rails has deprecated using return, break and throw to exit a transaction. The suggested solution is to add condition blocks.

https://blog.saeloun.com/2020/04/06/rails-disallow-return-break-and-throw-to-exit-transaction.html

This means that this cop needs to be manually silenced.

I suggest upgrading the cop to recognising and ignoring when inside a transaction.

**Comment by koic** [comment]
`Rails/TransactionExitStatement` cop will help with that.
https://docs.rubocop.org/rubocop-rails/cops_rails.html#railstransactionexitstatement
